src/Monticello/MonticelloBootstrap.class.st~

```diff
@@ -0,0 +1,170 @@
+"
```

Comment:
This is a temporary file. Can we get rid of it?

Reply:
yeup